gnat2goto/driver/tree_walk.adb

```diff
@@ -4365,12 +4367,9 @@ package body Tree_Walk is
    -------------------------
 
    procedure Do_Withed_Unit_Spec (N : Node_Id) is
-      Unit_Name : constant String := Get_Name_String (Get_Unit_Name (N));
    begin
-      if Defining_Entity (N) = Stand.Standard_Standard or else
-        Unit_Name = "system%s"
-      then
-         --  At the moment Standard or System are not processed: TODO
+      if Defining_Entity (N) = Stand.Standard_Standard then
+         --  At the moment Standard is not processed: TODO
```

Comment:
Could you raise a GitHub issue to cover this, and then reference the issue in the TODO comment here?

Reply:
#252 and also added to the comment.